sdk/src/logs/batch_log_record_processor.cc

```diff
@@ -208,6 +208,8 @@ void BatchLogRecordProcessor::Export()
       break;
     }
 
+    // Reserve space for number of records and avoid reallocation of vector.
```

Comment:
nit: make the spelling consistent between the 2 comments.

Reply:
Resolved comments.